packages/ledger/src/lib/ledger.spec.ts

```diff
@@ -57,13 +61,32 @@ const createLedgerWallet = async (deps: MockWalletDependencies = {}) => {
     };
   });
 
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { setupLedger } = require("./ledger");
+
+  const provider = mock<ProviderService>();
+  const storage = mock<JsonStorageService>({
+    getItem: jest.fn(async (key) => storageState[key] || null),
+    setItem: jest.fn(async (key, value) => {
+      // @ts-ignore
+      storageState[key] = value;
+    }),
+  });
+  const ledgerWallet = await mockWallet<HardwareWallet>(setupLedger(), {
+    storage,
+    provider,
+    ...deps,
+  });
+
   provider.viewAccessKey.mockResolvedValue({
     nonce: 0,
     permission: "FullAccess",
     block_height: 0,
     block_hash: "block_hash",
   });
 
+  provider.sendTransaction.mockResolvedValue(getSendTransactionResponse());
```

Comment:
I'm not sure we need to mock this method as it'll already be mocked by `jest-mock-extended` for the purpose of what we need as we don't really need to assert the return value in our tests as TypeScript helps us enforce this